modular_skyrat/modules/engineering_override/code/engineering_override.dm

```diff
@@ -0,0 +1,126 @@
+/**
+ * This module sets airlocks in certain areas to be able to have an Engineer Override on orange alert.
+ * Crew with ID cards with the engineering flag will be able to access these areas during those times.
+ */
+
+// Is this area eligible for engineer override?
+/area
+	var/engineering_override_eligible = FALSE
```

Comment:
First of all, if you have the comment block at the top formatted that way, AUTODOC will make it the documentation for all of `/area` which isn't what we want.

Secondly, this is how you document a variable properly, now you know!
```suggestion
// This module sets airlocks in certain areas to be able to have an Engineer Override on orange alert.
// Crew with ID cards with the engineering flag will be able to access these areas during those times.

/area
	/// Is this area eligible for engineer override?
	var/engineering_override_eligible = FALSE
```